src/common/chat-participants/powerpages/commands/create-site/CreateSiteHelper.ts

```diff
@@ -189,3 +197,70 @@ export async function collectSiteCreationInputs(siteName: string, envList: IEnvI
     // Return the collected site creation inputs including site name, environment name, and domain name
     return siteInputState;
 }
+
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+export async function populateSiteRecords(siteName: string, sitePagesList: string[], sitePages: any, orgUrl: string, telemetry: ITelemetry) {
+    return vscode.window.withProgress({
```

Comment:
Use [showProgressWithNotification](https://github.com/microsoft/powerplatform-vscode/blob/1af6cf58525eff52f5c019e445cc93b5dd12493f/src/common/utilities/Utils.ts#L107C1-L115C2). You can modify it to pass `progress` parameter